src/vl-typography.js

```diff
@@ -61,7 +61,15 @@ export class VlTypography extends vlElement(HTMLElement) {
     return observer;
   }
 
-  __processTemplateParameters(template, params) {
+  /**
+   * Dit vormt een template met placeholders voor parameters om in een tekst waarin deze placeholders vervangen
+   * zijn met hun waarden.
+   *
+   * @param {string} template De template waarin placeholders zitten in de vorm van "${parameter.parameterNaam}"
+   * @param {object} params Een object met keys die de naam van de parameter voorstellen (bv. "parameterNaam") en hun waarden
+   * @returns {string} De resulterende tekst
+   */
+  static processTemplateParameters(template, params) {
```

Comment:
replaceTemplateParameters klinkt voor mij iets beter, process is nogal algemeen en replace reflecteerd meer wat het doet

Reply:
Ik was zelf aan het twijfelen over `renderTemplate` (zo noemt het vaak in template engines), maar dat had dan ook weer een connotatie (visueel renderen).
Daarom had ik het voor nu maar gehouden op wat het was. Als de groep tot een voorstel komt wat iedereen ok vindt, wil ik dit gerust aanpassen.

Reply:
@coemans, jij een voorkeur?

Reply:
Het verleden leert me dat wanneer persoonlijke naamgeving voorkeuren/discussies de focus krijgen in een PR de rest van de code meestal zo uitstekend is dat het eigenlijk niet uitmaakt. Voor mij is `replaceTemplateParameters` ook prima. Het `renderTemplate` voorstel vind ik persoonlijk niet beter omdat het inderdaad lijkt dat er dan ook gevisualiseerd wordt.

Reply:
> de rest van de code meestal zo uitstekend

Ge moet me niet doen blozen, Tom :)

Ik zal het aanpassen naar replace, dan is iedereen content.